part-i.md

```diff
@@ -310,7 +310,7 @@ import * as HL from "./HelperLib";
 import { foo, bar } from "./SomeLib"
 
 const id = a => a
-const swap = [a, b] => [b, a]
```

Comment:
That's not valid TypeScript — running it would cause the JavaScript runtime to throw `Uncaught SyntaxError: Malformed arrow function parameter list`.